src/recommendations/RecommendationsPage.jsx

```diff
@@ -77,7 +89,7 @@ const RecommendationsPage = (props) => {
           <Image className="logo" alt={getConfig().SITE_NAME} src={getConfig().LOGO_URL} />
         </Hyperlink>
       </div>
-      {(!isLoading && recommendations.length === RECOMMENDATIONS_COUNT) ? (
+      {(!isLoading && recommendations.length >= RECOMMENDATIONS_COUNT) ? (
```

Comment:
why did you change operator from `===` to `>=`?

Reply:
I'll revert this. I was doing some testing.

Reply:
But I think its okay even if the condition is `>=`. I'll keep it like this. It just means we won't show anything less than 4 but over 4 is okay.

Reply:
We are doing slicing to 4 recs. That means the recs can never be more than 4. I know this condition works but i think in code semantics, it does not make sense.